modular_zubbers/code/modules/clothing/neck_wearing.dm

```diff
@@ -0,0 +1,105 @@
+/*
+Functional Toggle lets you convert stuff to functional (exo suit), with armor, cold and heat protection values, to non functional (neck), with all those set to zero.
+It allows people to use a jacket over a piece or armor and only sacrifice the minimal amount of functionality in the pursuit of design.
+Use CTRL + SHIFT + LEFT CLICK to turn them on and off.
+*/
+/obj/item/clothing/suit
+	/// When set to TRUE, this particular suit is not able to use the functional toggle
+	var/only_functional
+	/// temp list to restore the intitial functional values (for armor, cold protection, etc) to the state they were in prior to using the functional toggle
+	var/list/functional_suit_values
+
+/obj/item/clothing/suit/Initialize(mapload)
+	. = ..()
+	if(!(flags_1 & HAS_CONTEXTUAL_SCREENTIPS_1))
+		register_context()
+
+/obj/item/clothing/suit/examine(mob/user)
+	. = ..()
+	if(!only_functional)
+		. += span_info("Ctrl + Shift + Left Click to swap between functional (suit) and non-functional (neck) mode, to allow for things such as wearing a (nonfunctional) jacket over a piece of armor for the visual effect.")
+
+#define PREV_SLOT_FLAGS "fs_slots"
+#define PREV_COLD_PROTECTION "fs_cold"
+#define PREV_HEAT_PROTECTION "fs_heat"
+#define PREV_SLOWDOWN "fs_slow"
+#define PREV_ARMOR_DATUM "fs_armor"
+
+/obj/item/clothing/suit/click_ctrl_shift(mob/user)
+	if(!iscarbon(user))
+		return NONE
+	if(only_functional)
+		return NONE
+	var/mob/living/carbon/char = user
+	if((char.get_item_by_slot(ITEM_SLOT_NECK) == src) || (char.get_item_by_slot(ITEM_SLOT_OCLOTHING) == src))
+		to_chat(user, span_warning("You can't adjust [src] while wearing it!"))
+		return CLICK_ACTION_BLOCKING
+	if(!user.is_holding(src))
+		to_chat(user, span_warning("You must be holding [src] in order to adjust it!"))
+		return CLICK_ACTION_BLOCKING
+	if(slot_flags & ITEM_SLOT_OCLOTHING)
+		functional_suit_values = list(
+			PREV_SLOT_FLAGS = slot_flags,
+			PREV_COLD_PROTECTION = cold_protection,
+			PREV_HEAT_PROTECTION = heat_protection,
+			PREV_SLOWDOWN = slowdown,
+			PREV_ARMOR_DATUM = armor_type,
+		)
+		slot_flags = ITEM_SLOT_NECK
```

Comment:
Not a requirement, but slot_flags is a bitmask meaning it supports multiple flags, thus you could simply make all clothing ITEM_SLOT_NECK | ITEM_SLOT_OCLOTHING, and check on equipped(mob/user, slot, initial) if it's the neck, and make it nonfunctional then, circumventing the need to toggle it